### PR TITLE
[OT185-61] Contact list endpoint

### PR DESCRIPTION
**Pull request by locknet11**
## Summary
This PR adds: 
 - ContactController
 - ContactService
 - ContactServiceImpl

 
This PR modifies: 
 - Contact
 - ApiConstants
 - SecurityConfig

### Type of change
- [x] New feature
- [ ] New Tests
- [ ] Bug fix
- [ ] Refactor

### Checklist

- [x] Traceability between this change and Jira.
- [x] ```mvn clean install``` was run and all tests completed successfully.
- [x] There are no unused variables and/or imports in the modified classes.
- [x] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [x] Slf4j was used for the application log instead ```System.out```.
- [x] Public methods are documented with ```javadoc```.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/ports/input/rs/controller/ContactController.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@RestController
+@RequestMapping(CONTACTS_URI)
+@RequiredArgsConstructor
+public class ContactController {
+
+	private final ContactService contactService;
+
+	@GetMapping
+	@PreAuthorize("hasRole('ROLE_ADMIN')")
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/ports/input/rs/controller/ContactController.java]: ¿El PreAuthorize está funcionando? Porque debería ser `hasRole('ADMIN')`

**Review by manerajona — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V1.61__add_contacts.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO contact (contact_id, name, phone, email, message, created_at, is_active) VALUES
```
Comment (manerajona) [inline src/main/resources/db/migration/V1.61__add_contacts.sql]: Esto no lo subamos, por favor.

**Review by manerajona — CHANGES_REQUESTED** [review]

File: src/main/java/com/alkemy/ong/ports/input/rs/controller/ContactController.java (hunk 1 above)
Comment (locknet11) [reply src/main/java/com/alkemy/ong/ports/input/rs/controller/ContactController.java]: Listo!

File: src/main/resources/db/migration/V1.61__add_contacts.sql (hunk 2 above)
Comment (locknet11) [reply src/main/resources/db/migration/V1.61__add_contacts.sql]: Listo, ya lo saque

**Pull request merged**